pkg/apis/awsproviderconfig/v1alpha1/awsmachineproviderconfig_types.go

```diff
@@ -135,7 +135,11 @@ type AWSMachineProviderConfig struct {
 
 	// LoadBalancerNames is the names of the load balancers to which the new instance
 	// should be added once it is created.
-	LoadBalancerNames []string `json:"loadBalancerIds"`
+	LoadBalancerNames []string `json:"loadBalancerNames"`
```

Comment:
Would it make sense for this to be a common struct? e.g:
```
type loadBalancers struct {
	Names
	type
}
```

Reply:
Added a load balancer ref type  that has both name and type, ptal